trillian_log_api.proto

```diff
@@ -182,23 +183,42 @@ service TrillianLog {
 
     // No direct equivalent at the storage level
     rpc GetInclusionProof (GetInclusionProofRequest) returns (GetInclusionProofResponse) {
+      option (google.api.http) = {
+        get: "/v1beta1/log/{log_id}/size/{tree_size}/inclusion/index/{leaf_index}"
```

Comment:
Thanks for your review! 

Agreed that the API wasn't designed to be resource oriented. Hopefully someday we can add a set of resource oriented methods and deprecate the ones that aren't.  I created this PR mostly so I could debug the log easily without resorting to `grpc_cli`.  The methods here are most certainly alpha mode and can change.